pkg/controller/manager/manager_controller.go

```diff
@@ -483,22 +481,23 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 	}
 
 	managerCfg := &render.ManagerConfiguration{
-		KeyValidatorConfig:      keyValidatorConfig,
-		ESSecrets:               esSecrets,
-		TrustedCertBundle:       trustedBundle,
-		ESClusterConfig:         esClusterConfig,
-		TLSKeyPair:              tlsSecret,
-		PullSecrets:             pullSecrets,
-		Openshift:               r.provider == operatorv1.ProviderOpenShift,
-		Installation:            installation,
-		ManagementCluster:       managementCluster,
-		TunnelSecret:            tunnelSecret,
-		InternalTrafficSecret:   internalTrafficSecret,
-		ClusterDomain:           r.clusterDomain,
-		ESLicenseType:           elasticLicenseType,
-		Replicas:                replicas,
-		ComplianceFeatureActive: installCompliance,
-		UsePSP:                  r.usePSP,
+		KeyValidatorConfig:             keyValidatorConfig,
+		ESSecrets:                      esSecrets,
+		TrustedCertBundle:              trustedBundle,
+		ESClusterConfig:                esClusterConfig,
+		TLSKeyPair:                     tlsSecret,
+		PullSecrets:                    pullSecrets,
+		Openshift:                      r.provider == operatorv1.ProviderOpenShift,
+		Installation:                   installation,
+		ManagementCluster:              managementCluster,
+		TunnelSecret:                   tunnelSecret,
+		InternalTrafficSecret:          internalTrafficSecret,
+		ClusterDomain:                  r.clusterDomain,
+		ESLicenseType:                  elasticLicenseType,
+		Replicas:                       replicas,
+		Compliance:                     complianceCR,
```

Comment:
Why pass both Compliance and ComplianceLicenseFeatureActive? It seems like it would be fine to have one variable that represented if compliance is Enabled.

Reply:
I opted to pass the compliance CR presence and license feature status as two separate inputs since the Manager UI has a 'license insufficient' rendering state, where it renders the nav item but informs the user that they are not licensed to use a feature. This is separate from its 'feature flagged off' state, which omits the nav item.

Assuming your suggestion is one boolean input which is true when the CR is present and license feature is active: if we were to set the compliance feature flag on the Manager to false as a result of the license feature not being active, there would be no differentiation in the UI between the CR not being present and the license being insufficient. Hence why I opted to pass two separate inputs

WDYT? Let me know if I'm missing something.